dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/validation/EventImportValidationTest.java

```diff
@@ -118,6 +118,26 @@ void failValidationWhenTrackedEntityAttributeHasWrongOptionValue()
         assertHasOnlyErrors( importReport, ValidationCode.E1125 );
     }
 
+    @Test
+    void successWhenEventHasValidMultiTextValue()
```

Comment:
I think this duplicates tests unit tests in dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/validator/event/DataValuesValidatorTest.java

I would therefore remove this one in favor of the unit tests

Reply:
ok, removed.